gazebo_controller_init:
  propor: 100
  integr: 60
  deriv: 1
  init_legs_length: 0.3

stewart: 
  max_speed_end_eff: 3
  max_dis_from_goal: 1.3
  min_dis_from_goal: 0.25
  min_roll_angle_diff : 10
  min_pitch_angle_diff : 10
  min_yaw_angle_diff : 10
  min_speed_to_settle: 0.1
 
  x_range: [-1.2 , 1.5]  # tested on z = 1.2
  y_range: [-1.5 , 1.5]  # tested on z = 1.2
  z_range: [0.3 , 1.7]
  max_pitch_angle: 80  # platform limits are [-23, 23] deg tested on z = 1.2
  max_roll_angle: 80   # platform limits are [-23, 23] deg tested on z = 1.2
  max_yaw_angle: 80    # platform limits are [-45, 45] deg tested on z = 1.2

  p_range_value: [30,100] # [30,100]
  i_range_value: [10,60] # [10,60]
  d_range_value: [0,1]

  f1_range_value: [-100,100]
  f2_range_value: [-100,100]
  f3_range_value: [-100,100]
  f4_range_value: [-100,100]
  f5_range_value: [-100,100]
  f6_range_value: [-100,100]

  reach_goal_reward : 0
  go_very_far_reward: -1
  immobility_anxiety  : -0.001
  very_far_quadratic: -1000

reaching_task:
  reach_x: 0
  reach_y: 0
  reach_z: 1.2  
  reach_roll: 0
  reach_pitch: 0
  reach_yaw: 30